Project/src/MakeCall/VolumeVisualizer.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { LocalAudioStream } from '@azure/communication-calling';
+
+export default class VolumeVisualizer extends React.Component {
```

Comment:
Since this is a new control. It would have been a good idea to convert this in a functional component to avoid issues with `this` and it could allow the use of hooks as well